jobs/math_utils.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
```

Comment:
Can you try to optimize the code with numba, and check if it runs faster than with np.roots?